src/app/src/main/kotlin/org/imperial/mrc/hint/db/DbConfig.kt

```diff
@@ -18,4 +18,13 @@ class DbConfig {
         dataSourceBuilder.password(appProperties.dbPassword)
         return dataSourceBuilder.build()
     }
+
+    fun dataSource(url: String, user: String, password: String): DataSource {
```

Comment:
Codecov thinks this isn't under test. It is actually in the `beforeAll` method of a test suite, so as far as I'm concerned it is under test!